test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -154,7 +167,7 @@ Describe "PowerShellGet - Module tests (Admin)" -tags @('Feature', 'RequireAdmin
         $module = Get-Module $ContosoServer -ListAvailable
         $module.Name | Should be $ContosoServer
         $module.ModuleBase | Should Be $installedModuleInfo.InstalledLocation
-    }
+    } -Skip:$IsLinux
```

Comment:
Please use `-Pending` instead of `-Skip`. We track all pending tests because they are supposed to work but now at the time. `-Skip` means they are not applicable to the platform.

Reply:
And also, please add `-Pending` before the open curly bracket, like
```
It "Should install a module correctly to the required location with default AllUsers scope" -Pending:$IsLinux {
```